content/docs/ops/traffic-management/troubleshooting/index.md

```diff
@@ -82,7 +82,7 @@ To confirm there is a conflict, check whether the `STATUS` field in the output o
 is set to `CONFLICT` for your service. For example:
 
 {{< text bash >}}
-$ istioctl authn tls-check httpbin.default.svc.cluster.local
+$ istioctl authn tls-check istio-ingressgateway-6b454db869-5zn2j.istio-system httpbin.default.svc.cluster.local
```

Comment:
This is an actual bug in the current docs. It's an example of what we're trying to avoid with testing. The `istioctl authn tls-check` command syntax has changed to require an extra source pod argument, but this doc wasn't updated.

But, since the command requires the running instance name in the syntax, moving the command to a snippet wouldn't be a command that a test could run directly. The test would need to preprocess the snippet to change the pod name to the running one.